src/transformers/onnx/convert.py

```diff
@@ -438,10 +438,12 @@ def validate_model_outputs(
 
         # Values
         if not np.allclose(ref_value, ort_value, atol=atol):
+            bad_idxes = np.logical_not(np.isclose(ref_value, ort_value, atol=atol))
```

Comment:
nit: we usually prefer being explicit where possible with variable names
```suggestion
            bad_indices = np.logical_not(np.isclose(ref_value, ort_value, atol=atol))
```